website/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "devDependencies": {
     "@angular/cdk": "5.0.2",
-    "@angular/cli": "1.6.1",
+    "@angular/cli": "~1.6.1",
```

Comment:
Do we need to do this? I like the idea of pinning, because people where having issues when I first started to book. Things are certainly more stable now, but it still worries me a little.

Reply:
required update per https://github.com/angular/angular-cli/issues/9307

Reply:
We could pin the version to 1.6.8 instead?

Reply:
Ok, I like that idea.